src/pkg/scan/vuln/summary.go

```diff
@@ -21,13 +21,14 @@ import (
 // NativeReportSummary is the default supported scan report summary model.
 // Generated based on the report with v1.MimeTypeNativeReport mime type.
 type NativeReportSummary struct {
-	ReportID   string                `json:"report_id"`
-	ScanStatus string                `json:"scan_status"`
-	Severity   Severity              `json:"severity"`
-	Duration   int64                 `json:"duration"`
-	Summary    *VulnerabilitySummary `json:"summary"`
-	StartTime  time.Time             `json:"start_time"`
-	EndTime    time.Time             `json:"end_time"`
+	ReportID    string                `json:"report_id"`
+	ScanStatus  string                `json:"scan_status"`
+	Severity    Severity              `json:"severity"`
+	Duration    int64                 `json:"duration"`
+	Summary     *VulnerabilitySummary `json:"summary"`
+	CVEBypassed []string              `json:"-"`
```

Comment:
It may be even better to return the CVE-Bypassed in the API such that UI can highlight what vulnerabilities in the whitelist, but not a must for this PR.

Reply:
@reasonerjt 
You can raise an issue for this because it needs UI work. For the current case, it follows the previous behavior, only prints them in the log.